From 534ab75d357815b6a6e56ffbaf045a4ea2d1dea5 Mon Sep 17 00:00:00 2001
From: Andy Ross <andrew.j.ross@intel.com>
Date: Wed, 11 Jun 2014 20:01:22 -0700
Subject: [PATCH 067/117] platform_byt_regulator: Disable Rohm PMIC regulator

This was enabled globally, but should be probed off of the PMIC device
only on systems where it exists.

Issue: GMIN-1553
Signed-off-by: Andy Ross <andrew.j.ross@intel.com>
---
 arch/x86/platform/intel-mid/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/x86/platform/intel-mid/Makefile b/arch/x86/platform/intel-mid/Makefile
index 4e0b4b1..e0c381a 100644
--- a/arch/x86/platform/intel-mid/Makefile
+++ b/arch/x86/platform/intel-mid/Makefile
@@ -3,7 +3,6 @@ obj-$(CONFIG_EARLY_PRINTK_INTEL_MID) += early_printk_intel_mid.o
 
 obj-$(CONFIG_GMIN_INTEL_MID) += intel_mid_pcihelpers.o platform_imx134.o atomisp_gmin_platform.o
 obj-$(CONFIG_GMIN_INTEL_MID) += platform_ov2722.o
-obj-$(CONFIG_GMIN_INTEL_MID) += platform_byt_regulator.o
 
 obj-$(CONFIG_GMIN_INTEL_MID) += intel_mid_pcihelpers.o platform_imx134.o atomisp_gmin_platform.o
 
-- 
1.9.3

